mesheryctl/internal/cli/root/perf/list.go

```diff
@@ -233,7 +233,7 @@ func fetchPerformanceProfileResults(url string, profileID string) ([][]string, [
 	}
 	resp, err := client.Do(req)
 	if err != nil {
-		return nil, nil, err
+		return nil, nil, errors.New("outdated authentication token")
```

Comment:
"or invalid token"? In this flow, are you sure the token is outdated?

Reply:
I guess you're right, invalid token would be better here!